modules/sievefilters/site.js

```diff
@@ -130,13 +130,6 @@ var hm_sieve_possible_actions = function() {
             type: 'string',
             extra_field: false
         },
-        {
```

Comment:
we are using this filter at `https://github.com/cypht-org/cypht/blob/master/modules/sievefilters/modules.php#L915` 

I mean I don't understand why you are deleting this here.

Reply:
From what I understand, the value \$action->value passed into the addAction method is not explicitly an associative array in the current implementation. Could you please explain to me how this change might "break" sieve's behavior? It would help me better understand the issue and ensure that the implementation meets the expectations of PhpSieveManager. thanks